src/SFA.DAS.AssessorService.Web/Controllers/Apply/StandardController.cs

```diff
@@ -580,6 +589,11 @@ public async Task<IActionResult> OptOutStandardVersionConfirmation(string refere
             if (!standardVersions?.Any() ?? false)
                 throw new NotFoundException($"The standard reference {referenceNumber} cannot be found");
 
+            if (!ModelState.IsValid)
```

Comment:
This is going round in a circle - the model would not be invalid here anyway?

Reply:
Removed